lapis/src/main/kotlin/org/genspectrum/lapis/model/VariantQueryCustomListener.kt

```diff
@@ -141,8 +343,13 @@ class VariantQueryCustomListener(
         if (ctx == null) {
             return
         }
+        try {
+            referenceGenomeSchema.getGeneFromLowercaseName(ctx.geneOrName().text.lowercase()).name
+        } catch (e: RecognitionException) {
+            throw BadRequestException("Gene not implemented", e)
+        }
```

Comment:
Do we need this catch block?

Reply:
We need to catch if the gene is invalid - but perhaps `getGeneFromLowercaseName` will also handle this - in any case this error message needs to be improved!